### PR TITLE
Initial Project Scaffolding

### PR DESCRIPTION
**Pull request by AnAngryPlatapus**
Tom if u want some chedda, i need some cheese

### CODE REVIEW

File: tests/__pycache__/conftest.cpython-311-pytest-7.4.2.pyc
Comment (thomasaiman) [inline tests/__pycache__/conftest.cpython-311-pytest-7.4.2.pyc:1]: bonk

File: tests/__pycache__/test_statistics.cpython-311-pytest-7.4.2.pyc
Comment (thomasaiman) [inline tests/__pycache__/test_statistics.cpython-311-pytest-7.4.2.pyc:1]: womp

File: sunny.tech.ipynb [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    "asset_returns = \"\"\"\n",
+    "Date,Asset1,Asset2,Asset3,Asset4\n",
+    "2023-01-01,0.02,0.03,0.015,0.01\n",
+    "2023-01-02,0.015,0.025,0.012,0.008\n",
+    "2023-01-03,-0.01,-0.02,-0.005,-0.003\n",
+    "2023-01-04,0.03,0.035,0.018,0.015\n",
+    "2023-01-05,0.012,0.02,0.01,0.007\n",
+    "\"\"\"\n",
+    "\n",
+    "from pandas"
```
Comment (thomasaiman) [inline sunny.tech.ipynb:148]:
```suggestion
    "import pandas as pd"
```

**Review by thomasaiman — CHANGES_REQUESTED** [review]
prolly .gitignore `**.pyc`

File: tests/__pycache__/test_statistics.cpython-311-pytest-7.4.2.pyc
Comment (AnAngryPlatapus) [reply tests/__pycache__/test_statistics.cpython-311-pytest-7.4.2.pyc:1]: yup its gone

**Pull request merged**